pkg/resources/function.go

```diff
@@ -192,12 +240,12 @@ func CreateFunction(d *schema.ResourceData, meta interface{}) error {
 		builder.WithImports(imports)
 	}
 
-	// handler for Java
+	// handler for Java / python
 	if v, ok := d.GetOk("handler"); ok {
 		builder.WithHandler(v.(string))
 	}
 
-	// target path for Java
+	// target path for Java / python
```

Comment:
Python should be capitalized

Reply:
@sfc-gh-kmaurya  please don't ignore feedback from senior engineers